From a615a242054d02ec41a2ad9b2b49ebf542f4badc Mon Sep 17 00:00:00 2001
From: Jenna-66 <jenna@jennalody.de>
Date: Tue, 23 Dec 2025 17:31:11 +0100
Subject: [PATCH 7/7] !fixup: Make su domain permissive on user builds

Signed-off-by: Jenna-66 <jenna@jennalody.de>
---
 contexts/Android.bp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/contexts/Android.bp b/contexts/Android.bp
index 842e3181d..dcaf3b4ad 100644
--- a/contexts/Android.bp
+++ b/contexts/Android.bp
@@ -434,6 +434,7 @@ se_policy_binary {
     name: "plat_precompiled_sepolicy",
     srcs: [":plat_sepolicy.cil"],
     installable: false,
+    permissive_domains_on_user_builds: ["su"],
 }
 
 // Use plat_precompiled_sepolicy instead of precompiled_sepolicy because
-- 
2.53.0

